src/main/scala/org/ergoplatform/nodeView/wallet/ErgoWalletActor.scala

```diff
@@ -68,7 +67,10 @@ class ErgoWalletActor(settings: ErgoSettings,
   // The state context is being updated by listening to state updates.
   private def stateContext: ErgoStateContext = storage.readStateContext
 
-  private var height: Int = ErgoHistory.GenesisHeight
+  /**
+    * Height of full-blocks chain as reported by the state processor
+    */
+  private var fullHeight: Int = ErgoHistory.EmptyHistoryHeight
```

Comment:
The name is not descriptive enough. What the `full` part means?

Reply:
@aslesarenko it means height of the full-blocks chain (not the headers-chain), actually, it is already written in the comment above.